kubetest/extract_k8s.go

```diff
@@ -69,7 +69,7 @@ func (l *extractStrategies) String() string {
 func (l *extractStrategies) Set(value string) error {
 	var strategies = map[string]extractMode{
 		`^(local)`:                    local,
-		`^gke-?(staging|test)?$`:      gke,
+		`^gke-?(default|latest)?$`:    gke,
```

Comment:
@mindprince we'll keep `--extract=gke` for now for backwards compatibility. Added a warning, and will move to gke-default when the kubetest is published

Reply:
Can you also update all the jobs using `gke` to use `gke-default`?